app/services/os-datastore.js

```diff
@@ -493,3 +498,20 @@ module.exports.publish = function(descriptor, options) {
       });
   });
 };
+
+module.exports.isDataStoreUrl = function(urlToCheck) {
+  if (!_.isString(urlToCheck)) {
+    return false;
+  }
+  var parsed = url.parse(urlToCheck);
+  if (!_.isObject(parsed)) {
+    return false;
+  }
+
+  var checkHost1 = parsed.hostname == 's3.amazonaws.com';
```

Comment:
I was sure I answered you in Slack, now I see it didn't send...
Anyway, I'll add an API for that in the conductor (as datastore URL is environment dependent and also depends on the user id).